src/plugins/github/README.md

```diff
@@ -0,0 +1,175 @@
+# GitHub Plugin
+
+This is the SourceCred GitHub plugin. It creates a cred graph based on data
+from [GitHub].
+
+[GitHub]: https://github.com/
+
+## Nodes
+
+The GitHub plugin creates the following kinds of nodes:
+
+### Repo
+
+A GitHub repository, e.g. [sourcecred/sourcecred]. It will be directly
+connected to all of the pulls and issues in the repository. Setting a weight on
+the repository will have no effect (i.e. they do not mint cred). This will
+change when we switch to [CredRank].
+
+[CredRank]: https://github.com/sourcecred/sourcecred/issues/1686
+
+[sourcecred/sourcecred]: https://github.com/sourcecred/sourcecred
+
+### Issue
+
+A GitHub issue, e.g. [sourcecred/sourcecred#40]. Issues are connected to their
+author(s), to entities that they reference, to their comments, and to the
+containing GitHub repository. If issues are given a positive weight, then each
+issue will mint cred. Note that this could be abused by spammers.
```

Comment:
It's helpful to reference why this can be abused. Even if it's without a link.
> This would create activity-minted cred. Which might be abused by spammers.

Ideally activity-minted cred has a link.